### DISCUSSION TITLE
bug: iOS firebase crash due to inavalid GOOGLE_APP_ID

### DISCUSSION THREAD

**Discussion by kishan-coditas** [ionic-team/capacitor#3652] [Q&A]
## Bug Report
Application crash with below error

`*** Terminating app due to uncaught exception 'com.firebase.core', reason: 'Configuration fails. It may be caused by an invalid GOOGLE_APP_ID in GoogleService-Info.plist or set in the customized options.'`

I try to google and there are lots of options and fixs I have tried that all but still I am not able get rid of this error.

I am using latest xcode version and my OS is "Big Sur"
### Capacitor Version
💊   Capacitor Doctor  💊 

Latest Dependencies:

  @capacitor/cli: 2.4.2
  @capacitor/core: 2.4.2
  @capacitor/android: 2.4.2
  @capacitor/electron: 2.4.2
  @capacitor/ios: 2.4.2

Installed Dependencies:

  @capacitor/cli 2.2.1
  @capacitor/ios 2.2.1
  @capacitor/android 2.2.0
  @capacitor/core 2.2.1
  @capacitor/electron not installed

[success] Android looking great! 👌
  Found 24 Capacitor plugins for ios:
    @capacitor-community/facebook-login (1.0.0)
    cordova-plugin-androidx (1.0.2)
    cordova-plugin-androidx-adapter (1.1.1)
    cordova-plugin-appavailability (0.4.2)
    cordova-plugin-apprate (1.5.0)
    cordova-plugin-browsertab (0.2.0)
    cordova-plugin-buildinfo (4.0.0)
    cordova-plugin-compat (1.2.0)
    cordova-plugin-customurlscheme (4.4.0)
    cordova-plugin-dialogs (2.0.2)
    cordova-plugin-enable-multidex (0.2.0)
    cordova-plugin-firebasex (9.1.2)
    cordova-plugin-globalization (1.11.0)
    cordova-plugin-googleplus (8.4.0)
    cordova-plugin-inappbrowser (3.2.0)
    cordova-plugin-ionic-webview (4.1.3)
    cordova-plugin-nativestorage (2.3.2)
    cordova-plugin-whitelist (1.3.4)
    cordova-plugin-x-socialsharing (5.6.4)
    cordova-sqlite-storage (5.0.0)
    cordova-universal-links-plugin (1.2.1)
    es6-promise-plugin (4.2.2)
    cordova-plugin-ionic-webview (4.1.3)
    cordova-plugin-whitelist (1.3.4)
[success] iOS looking great! 

```
PASTE OUTPUT HERE
```

### Platform(s)
iOS only android works great.

### Current Behavior
I am trying to build ios application I have followed all steps which is mentioned on the doc here(https://capacitorjs.com/docs/guides/push-notifications-firebase)

when I run the app on simulator it crashed with above mentioned error.

### Expected Behavior
Application should build :)

### Other Technical Details
I am using vscode as IDE and here are the other plugin which I am using along with firebasex.

`npm --version` output:
6.9.0
`node --version` output:
v10.16.3
`pod --version` output (iOS issues only):
1.9.1

**Comment by jcesarmobile** [comment]
It’s probably some incompatibility problem of one of your plugins, hard to know with the information you provided, not a capacitor bug, moving to discussion
